ci/kind/test-netpol-v2-conformance-kind.sh

```diff
@@ -141,11 +143,14 @@ function run_test {
   export KUBE_CONTAINER_RUNTIME_ENDPOINT=unix:///run/containerd/containerd.sock
   export KUBE_CONTAINER_RUNTIME_NAME=containerd
 
-  # TODO: use https://github.com/kubernetes-sigs/network-policy-api when conformance test config timeout and go dependency is fixed
-  git clone https://github.com/Dyanngg/network-policy-api.git
+  git clone https://github.com/kubernetes-sigs/network-policy-api
   pushd network-policy-api/conformance
   go mod download
-  go test -v --debug=true -timeout=15m
+  go test -v -run TestConformanceProfiles -args --conformance-profiles=AdminNetworkPolicy,BaselineAdminNetworkPolicy \
+    --supported-features=AdminNetworkPolicyNamedPorts,BaselineAdminNetworkPolicyNamedPorts \
+    --organization=antrea-io -project=antrea -url=https://github.com/antrea-io/antrea -version=v2.0 \
+    --additional-info=https://github.com/antrea-io/antrea/actions/workflows/kind.yml \
```

Comment:
I'm curious to know what these 4 flags are for?

Reply:
So that we can generate a conformance report and put it upstream like https://github.com/kubernetes-sigs/network-policy-api/blob/main/conformance/reports/v0.1.2/ovn-kubernetes.yaml

Reply:
How do you want to handle the version though? Because technically, the version being tested here is unreleased when this is run as part of CI.

What you could do if you want is use an environment variable, that can be set when calling this script, and would otherwise default to the contents of the `VERSION` file (which is at the root of the repository). Then you would add a new Github workflow that can be run on demand (`workflow_dispatch` trigger) and would run the tests for a specific released Antrea version (version string to be provided as a workflow input). The workflow would upload the generated compliance report as an artifact for convenient download, and could even be run automatically for new Antrea releases.
I think that's the right approach. but it could be done in a later PR. For now I would just recommend implementing part of this proposal:
- add VERSION environment variable or command-line flag
  - if provided, install the specified Antrea version and use the provided version value as `-version`
  - if not provided, keep the current behavior, but use the contents of the `VERSION` file (`$(head -n1 $ROOT_DIR/VERSION)`) for `-version`

Reply:
Will do. I'll update this PR once the network-policy-api cuts a v2.0 release, which is going to happen very soon